addons/l10n_ir_accounting/data/account.account.template.csv

```diff
@@ -0,0 +1,389 @@
+"id","name","code","user_type_id/id","chart_template_id/id","reconcile"
+"account_account_template_0_111001","صندوق ریالی","111001","account.data_account_type_liquidity","l10n_ir_accounting.l10n_ir_chart_template","1"
```

Comment:
We have a convention that all the information should be written in english and then translated back to the desired language in the po files

Reply:
Also by convention in the column reconcile we use "True" or "False" and not numeric representation of a boolean